### ISSUE TITLE
inconsistency with paper

### ISSUE THREAD

**Issue by willleeney** [qcydm/VGAER#4]
the current code runs with B not concat(B,X) as detailed in the paper, as it is currently commented out. 

Also the kl divergence loss in line 141 doesn't need the logits.size(0) division as you are already taking the mean.
